src/test/java/org/xstefank/model/PersistentListTest.java

```diff
@@ -0,0 +1,39 @@
+package org.xstefank.model;
+
+import java.io.File;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xstefank.TestUtils;
+
+public class PersistentListTest {
+
+    private final String USERNAME = "testUser";
+    private final String FILE_NAME = "testList.txt";
+
+    private File userListFile = new File(TestUtils.TARGET_DIR, FILE_NAME);
+    private PersistentList testUserList;
+
+    @Before
+    public void before() {
+        testUserList = new PersistentList(TestUtils.TARGET_DIR, FILE_NAME);
```

Comment:
I don't think this is doing what you've intended - you create userListFile on line 15, and create PersistentList from it before every test. But `@After` deletes userListFile after first test so second test will create a new file at the same place by the call to PersistentList constructor - it's just no consistent so please clarify if this is what you've intended